index.bs

```diff
@@ -162,13 +165,26 @@ The <dfn attribute for="XRInputSource">gamepad</dfn> attribute is a {{Gamepad}}
  - More than one button
  - One or more axes
 
-Note: {{XRInputSource}}s in an {{XRSession}}'s {{XRSession/inputSources}} array are "live". As such values within them, such as the {{gamepad}} button and axis state, are updated in-place. This means that it doesn't work to save a reference to an {{XRInputSource}}'s {{gamepad}} on one frame and compare it to the same {{XRInputSource}}'s {{gamepad}} from a subsequent frame to test for state changes, because they will be the same object. Therefore developers that wish to compare input state from frame to frame should copy the state in question.
-
 XRSession {#xrsession-interface}
 ------------
 
 {{XRInputSource}}s are reported in the {{XRSession/inputSources}} array as they are connected and disconnected. When the presence of a {{XRInputSource/gamepad}} changes for any entry in the {{XRSession/inputSources}} array, the user agent MUST invoke the WebXR Device API's algorithm for [=change input source|responding to input source attribute changes=].
 
+XRFrame {#xrframe-interface}
+------------
+
+The [=XRFrame/apply frame updates=] steps are modified to include [=apply gamepad frame updates=].
```

Comment:
Is there anything we should be doing (perhaps in the other PR) to indicate that steps from other spec documents shouldn't have any ordering dependencies on each other?

Reply:
I can add a blurb to the other PR.